.env.example

```
@@ -17,4 +17,16 @@ ZAMMAD_HTTP_TOKEN=your-api-token-here
+
+# === Transport Configuration ===
+
+# Transport type: stdio (default) or http
+# MCP_TRANSPORT=stdio
+
+# HTTP Transport Settings (required if MCP_TRANSPORT=http)
+# Host to bind to (default: 127.0.0.1)
+# MCP_HOST=127.0.0.1
+
+# Port to listen on (required for HTTP transport)
+# MCP_PORT=8000
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)